browser/net/url_context.h

```diff
@@ -102,7 +102,9 @@ struct BraveRequestInfo {
   GURL ipfs_gateway_url;
   bool ipfs_auto_fallback = false;
 
-  bool ShouldMockRequest() const { return !mock_data_url.empty(); }
+  bool ShouldMockRequest() const {
```

Comment:
minor: https://github.com/brave/brave-core/blob/master/browser/net/brave_proxying_url_loader_factory.cc#L353 can be shortened with this as well

Reply:
I don't think that can be shortened - there's other logic after the redirect-specific part